zipline/pipeline/common.py

```diff
@@ -1,6 +1,7 @@
 """
 Common constants for Pipeline.
 """
+ACTUAL_VALUE_FIELD_NAME = 'actual_value'
```

Comment:
Is actual value the agreed upon name? This is the latest known metric (eps/revenue) value, right?